engine/src/main/java/com/redhat/cloud/notifications/recipients/itservice/pojo/response/ITUserResponse.java

```diff
@@ -0,0 +1,11 @@
+package com.redhat.cloud.notifications.recipients.itservice.pojo.response;
+
+import java.util.List;
+
+public class ITUserResponse {
```

Comment:
It would be better if all classes from `com.redhat.cloud.notifications.recipients.itservice.pojo.response` were annotated with `@JsonIgnoreProperties(ignoreUnknown = true)`. On the other hand, that annotation isn't really useful in the `.request` package.

Reply:
i remember I did that already. It probably went away during merge.

Reply:
added in #1044